src/handbook/peopleops/hiring/screening-call.md

```diff
@@ -18,6 +18,10 @@ During this call FlowFuse should understand:
 1. Alignment to [Values](/handbook/company/values.md).
 1. The candidate is enthusiastic about the company and the open role.
 1. The candidate is seeking full-time employment (FlowFuse only offers full-time positions).
+ * To get that information and make the candidate feel confortable use ice-braker questions like:
+   * "What did you think of the job posting?/is this what you're looking for in your next career move?"
+   * "I'd love to hear about your journey, especially the last 5-10 years. What led you here and what have you learned along the way?"
+   * "Tell me about your ideal role. What elements are non-negotiable for you to feel fulfilled professionally?"
```

Comment:
Should this list be moved to the top?

```suggestion
1. To get that information and make the candidate feels comfortable, use ice-braker questions like:
   * "What did you think of the job posting?/is this what you're looking for in your next career move?"
   * "I'd love to hear about your journey, especially the last 3-5 years. What led you here and what have you learned along the way?"
```

The non-negotiables I think is not really 'ice-breaker'. It's quite personal right away. I think the second question I won't use either for the same reason.